src/components/AdministrationPage/AdministrationPage.tsx

```diff
@@ -38,18 +44,48 @@ export const AdministrationPage = () => {
   };
 
   const [announcementMessage, setAnnouncementMessage] = useState<string>('');
+  const [etaTime, setEtaTime] = useState<string>();
+  const [isEtaNa, setIsEtaNa] = useState<boolean>(false);
+  const [announcementTouched, setAnnouncementTouched] = useState<boolean>(false);
+  const [etaTouched, setEtaTouched] = useState<boolean>(false);
   const dataContainerAnnouncement = useDataContainer(
     ({ serviceData }: IService<string>) => genericSettingsService.setAnnouncementBanner(serviceData as string),
     {
       initLoadingState: false,
     }
   );
 
+  const validateForm = () => {
+    setAnnouncementTouched(true);
+    setEtaTouched(true);
+    return !isAnnouncementInvalid() && !isEtaTimeInvalid();
+  };
+
+  const isAnnouncementInvalid = () => {
+    return isMaintenanceModeOn && announcementTouched && (!announcementMessage || announcementMessage === '');
+  };
+
+  const isEtaTimeInvalid = () => {
+    return isMaintenanceModeOn && etaTouched && !isEtaNa && (!etaTime || etaTime === '');
+  };
+
   useEffect(() => {
     genericSettingsService
       .getAnnouncementBanner()
       .then((response: any) => {
-        setAnnouncementMessage(response.data.banner);
+        const rawAnnouncement: string = response.data.banner;
+        const rawAnnouncementSet: Array<string> = rawAnnouncement ? rawAnnouncement.split(', ETA: ') : [];
+        setAnnouncementMessage(rawAnnouncementSet[0]);
+        if (rawAnnouncementSet[1]) {
+          if (rawAnnouncementSet[1] !== N_A) {
+            setEtaTime(rawAnnouncementSet[1]);
+            setIsEtaNa(false);
+          } else {
+            setEtaTime(undefined);
+            setIsEtaNa(true);
+          }
+        }
+        setIsMaintenanceModeOn(rawAnnouncementSet[1] ? true : false);
```

Comment:
Condition `rawAnnouncementSet[1] !== undefined` or even just `rawAnnouncementSet[1] ` would suffice I think.

Reply:
This is to include `rawAnnouncementSet[1] === null` as false as well.
Also rawAnnouncementSet[1] will not work:
![image](https://user-images.githubusercontent.com/6870455/203680010-58e27687-0b10-4413-ab12-1b47fad4f0c8.png)

Reply:
then consider using `!!rawAnnouncementSet[1]`

Reply:
updated accordingly, Thanks Martin!